src/app/config/config.dev.json

```diff
@@ -8,6 +8,7 @@
   "instant-counts-update-1": false,
   "instant-counts-update-2": false,
   "instant-counts-update-3": false,
+  "include-data-table": true,
   "authentication-service-type": "transmart",
   "oidc-server-url": "http://localhost:8081/auth/realms/master",
```

Comment:
I know it's not a change from this PR, but I have not noticed this in the previous one, duplicated `oidc-server-url` config and also the first one is invalid, missing `/protocol/openid-connect` part of the url.

Reply:
indeed, I did not notice this one. you mean the dev.config should be sth. like this?

```javascript
{
  "api-url": "http://localhost:8080",
  "api-version": "v2",
  "app-url": "http://localhost:4200",
  "autosave-subject-sets": false,
  "export-data-view": "default",
  "show-observation-counts": true,
  "instant-counts-update-1": false,
  "instant-counts-update-2": false,
  "instant-counts-update-3": false,
  "include-data-table": true,
  "authentication-service-type": "transmart",
  "oidc-server-url": "http://localhost:8081/auth/realms/master",
  "oidc-client-id": "glowingbear-js"
}
```

Reply:
what is the correct url for oidc-server-url?

Reply:
"oidc-server-url": "http://localhost:8080/auth/realms/master/protocol/openid-connect"